src/sentry/integrations/slack/integration.py

```diff
@@ -81,6 +92,61 @@ def get_config_data(self) -> Mapping[str, str]:
         )
         return {"installationType": metadata_.get("installation_type", default_installation)}
 
+    def get_organization_config(self) -> Sequence[tuple[str, bool]]:
+        """
+        Not sure why the base class is restricted to a sequence type, doesn't really make sense, however, it is
+        sufficient to our needs for now, and we can utilize it to return toggleable flags/feature on the integration
+        """
+
+        # Specifically using the parent method because the overwritten method on current class is hacked for another
+        # purpose at the integration/provider wide level, which is wrong/incorrect
```

Comment:
The current class is doing something at installation level not provider wide level. For example for one org and slack integration you can have one workspace installation with `classic_bot` and one with `workspace_app`. No? The value cannot change per installation like your toggle does but it seems installation level.

Reply:
followed the pattern in above comment ^